app/src/main/res/values-ar/strings.xml

```diff
@@ -101,6 +101,7 @@
         <b>Reddit:</b>
         https://www.reddit.com/r/SimpleMobileTools
     </string>
+    <string name="remove_done_items">Remove done items</string>
```

Comment:
move this new string to Checklists section please, not to the bottom

Reply:
Done